cicd/gitops/installing-openshift-gitops.adoc

```diff
@@ -9,6 +9,19 @@ toc::[]
 [role="_abstract"]
 {gitops-title} uses Argo CD to manage specific cluster-scoped resources, including cluster Operators, optional Operator Lifecycle Manager (OLM) Operators, and user management.
 
+[discrete]
+== Prerequisites
+
+* Access to the {product-title} web console.
```

Comment:
```suggestion
* You have access to the {product-title} web console.
```
Per the ISG, `Make list items grammatically parallel. For example, do not use a mix of phrases and sentences or a mix of passive and active voice in the same list."